src/juju/api.test.ts

```diff
@@ -558,35 +557,13 @@ describe("Juju API", () => {
       );
       expect(dispatch).toHaveBeenCalledWith(
         jujuActions.updateModelInfo({
-          modelInfo: {
-            ...abc123,
-            results: [
-              {
-                ...abc123.results[0],
-                result: {
-                  ...abc123.results[0].result,
-                  "agent-version": "2.3.4",
-                },
-              },
-            ],
-          },
```

Comment:
The `"agent-version"` type was fixed on the jujulib side so we no longer have to do these merges.